code/modules/surgery/blood_filter.dm

```diff
@@ -62,11 +61,14 @@
 		for(var/blood_chem in target.reagents.reagent_list)
 			var/datum/reagent/chem = blood_chem
 			target.reagents.remove_reagent(chem.type, min(chem.volume * chem_purge_factor, 10)) //Removes more reagent for higher amounts
-		if(tox_heal_factor > 0)
-			if(tox_loss <= 2)
-				target.setToxLoss(0)
-			else
-				target.adjustToxLoss(-(tox_loss * tox_heal_factor), forced=TRUE) //forced so this will actually heal oozelings too
+		if(tox_loss<=2)
+			target.setToxLoss(0) // To prevent chat spam over 1 toxy damage
```

Comment:
```suggestion
```

This and the if statement above it are not necessary, they do almost nothing and just serve to clutter the code.

I know these aren't your additions but this is a good time to remove them